server/base/src/main/java/org/apache/accumulo/server/security/SecurityUtil.java

```diff
@@ -93,4 +95,41 @@ public static String getServerPrincipal(String configuredPrincipal) {
       throw new RuntimeException("Could not convert configured server principal: " + configuredPrincipal, e);
     }
   }
+
+  /**
+   * Start a thread that periodically attempts to renew the current Kerberos user's ticket.
+   *
+   * @param ugi
+   *    The current Kerberos user.
+   * @param renewalPeriod
+   *    The amount of time between attempting renewals.
+   */
+  static void startTicketRenewalThread(final UserGroupInformation ugi, final long renewalPeriod) {
+    Thread t = new Daemon(new LoggingRunnable(renewalLog, new Runnable() {
```

Comment:
`SecuityUtil#serverLogin(AccumuloConfiguration)` aborts quickly if we're not actually running with Kerberos. So, no, this thread already won't run if the servers aren't configured to run with Kerberos.